test/specs/test-error-list.el

```diff
@@ -137,9 +137,8 @@
       (it "has the error ID in the 4th cell"
         (expect (aref cells 3) :to-equal
                 (list "W1"
-                      'type 'flycheck-error-list
-                      'face 'flycheck-error-list-id
-                      'help-echo "W1")))
```

Comment:
Alright, I just reverted to show the error ID as the help-echo text and reverted the test as well.